src/app/components/home-app/home-app.component.scss

```diff
@@ -67,16 +63,16 @@
   text-align: right;
 }
 
-.top-row {
-  width: 100%;
-  border-bottom-width: 1px;
-  border-bottom-style: solid;
-  padding: 0 24px;
-  box-sizing: border-box;
-  height: 35px;
-  display: flex;
-  align-items: center;
-}
+// .top-row {

```

Comment:
Go ahead and delete the commented out lines.  It is good practice not to check in commented out code.